release notes/v0.46.0.md

```diff
@@ -82,9 +82,17 @@ if (__ITER === 0) {
 
 Thanks @chrismoran-mica for contribution 🙇‍♂️.
 
-### `<big_feature_1>` `#pr`
+### Cloud Output v2 [#3117](https://github.com/grafana/k6/issues/3117)
 
-_what, why, and what this means for the user_
+After years of great service, we decided to refresh the k6 Cloud output introducing a more efficient end-to-end solution for ingesting the generated tests' metrics.
+
+The new output reduces the load generators' used resources for tests that produce many metrics. There is no significant difference in the user experience; it's expected to be the same. 
+
+The one thing worth highlighting is that the new output is strict about tags and drops tags if they are reserved. It's `test_run_id` and any tag that starts with two underscores, `__` (Prometheus convention).
```

Comment:
```suggestion
The one thing worth highlighting is that the new output is strict about tags and drops tags if they are reserved:
 - `test_run_id` as it is reserved for internal k6 cloud operations
 - any tag with a key that starts with two underscores (`__`), that is marked by Prometheus convention as reserved.
```
@olegbespalov WDYT? Could be an improvement?

Reply:
It's fine too me :+1: